packages/main/src/ColorPalettePopover.ts

```diff
@@ -85,11 +85,11 @@ type ColorPalettePopoverItemClickEventDetail = ColorPaletteItemClickEventDetail;
 })
 /**
  * Fired when the "open" state changes due to user interaction.
```

Comment:
Fired when the <code>ui5-color-palette-popover</code> is closed due to user interaction.